service/controller/cluster.go

```diff
@@ -48,7 +48,6 @@ import (
 	"github.com/giantswarm/aws-operator/service/controller/resource/s3bucket"
 	"github.com/giantswarm/aws-operator/service/controller/resource/secretfinalizer"
 	"github.com/giantswarm/aws-operator/service/controller/resource/service"
-	"github.com/giantswarm/aws-operator/service/controller/resource/snapshotid"
```

Comment:
The snapshotid handler is this still necessary for the control plane? or is also droppable?

Reply:
as written here https://github.com/giantswarm/aws-operator/pull/2596#issue-449306245

due to the old cluster having it in CF stack we cannot simply drop it

Reply:
I was not reading it carefully enough